os/drivers/lcd/Kconfig

```
@@ -139,12 +140,18 @@ config LCD_ST7701SN
 	---help---
 		Enable LCD Driver, ST7701SN with mipi.
 
-config LCD_ST7785
-	bool "ST7785 LCD Driver"
+config LCD_ST7785_TYPEA
+	bool "ST7785 Type A LCD Driver"
 	select LCD_INTERFACE_MIPI
 	---help---
 		Enable LCD Driver, ST7785 with mipi.
 
+config LCD_ST7785_TYPEB
+        bool "ST7785 Type B LCD Driver"
+        select LCD_INTERFACE_MIPI
+        ---help---
+                Enable LCD Driver, ST7785 with mipi.
+
```

Keepping LCD_ST7785 is better becase mipi_lcd.c can see only LCD_ST7785. 🤔
And add additional LCD_ST7785A LCD_ST7785B

```suggestion
config LCD_ST7785
	bool "ST7785 LCD Driver"
	select LCD_INTERFACE_MIPI
	---help---
		Enable LCD Driver, ST7785 with mipi.

if LCD_ST7785

config LCD_ST7785_TYPEA
	bool "ST7785 Type A LCD Driver"
	---help---
		Enable LCD Driver, ST7785 with mipi.

config LCD_ST7785_TYPEB
        bool "ST7785 Type B LCD Driver"
        ---help---
                Enable LCD Driver, ST7785 with mipi.
```